api/db/database.py

```diff
@@ -28,8 +29,11 @@ def get_db_engine(test_mode: bool = False):
             return create_engine(
                 DATABASE_URL, connect_args={"check_same_thread": False}
             )
-    elif DB_TYPE == "postgresql":
-        DATABASE_URL = f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
+    else:
+        if DB_DRIVER:
+            DATABASE_URL = f"{DB_TYPE}+{DB_DRIVER}://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
+        else:
+            DATABASE_URL = f"{DB_TYPE}://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
```

Comment:
You are doing the same thing

Reply:
I thought It might be more portable in this manner, but perhaps it's best not to alter that section.

Reply:
Should I revert this part ?